packages/kit/test/apps/basics/test/test.js

```diff
@@ -262,10 +262,10 @@ test.describe('Load', () => {
 			const payload_b = '{"status":200,"statusText":"","headers":{},"body":"Y"}';
 			// by the time JS has run, hydration will have nuked these scripts
 			const script_contents_a = await page.innerHTML(
-				'script[data-sveltekit-fetched][data-url="/load/serialization-post.json"][data-hash="3t25"]'
+				'script[data-sveltekit-fetched][data-url="/load/serialization-post.json"][data-hash="4jrjph"]'
```

Comment:
I still didn't figure out why this test is failing. I updated the hash to consider headers but it seems that the hash now it's different than the test above so maybe there is some header that I'm not considering when I hash manually.

Reply:
I found the issue, I was concatenating an ArrayBufferView + string.

To achieve this single hash computation I will need to update the hash function, I don't know if you want to do that. The other option is to concatenate the two hashes `hash(headers) + hash(body)`